rs-e2e/src/main/resources/assertion_definitions.json

```diff
@@ -101,6 +101,16 @@
         "OBX-3.1 = '57721-3'",
         "OBX.count() = 1"
       ]
+    },
+    {
+      "name": "LADPH ORU requirements",
+      "conditions": [
+        "MSH-10 = '010'"
```

Comment:
Instead of using the MSH-10 value, the conditions here could be:
```
"MSH-9.2 = 'R01'",
"MSH-5.2 = '2.16.840.1.114222.4.3.26.3.2'"
```
That means the assertion will be made for any LA ORU

Reply:
I mainly used `MSH-10 = '010'` because I like how it targets that specific sample file. I don't know how I feel about mixing and matching different test files and conditions for assertions. Maybe this can change once we have a better idea of all of the LA transformation.

Reply:
Sounds good